envi/archs/arm/disasm.py

```diff
@@ -2083,27 +2140,30 @@ def getOperAddr(self, op, emu=None):
         # there are certain circumstances where we can survive without an emulator
         pubwl = self.pubwl >> 3
         u = pubwl & 1
-
+        tsize = self.tsize # to help cope with ldcl
```

Comment:
Should have been updated in latest commit in this request. I basically disabled my code for now but left it in place until after the merge for reference. I tried to merge your latest into my code but it gets hard to tell what code you added and what code was there before. I have tried 3 different times and something gets screwed up each time. I did a rebase and it showed me the code that was different but that didn't end well either. So I relented to disabling my stuff and committing that and will test after merge?